app/helpers/application_form_helper.rb

```diff
@@ -1,6 +1,13 @@
+# frozen_string_literal: true
+
 module ApplicationFormHelper
   def application_form_full_name(application_form)
-    "#{application_form.given_names} #{application_form.family_name}"
+    if application_form.given_names.present? ||
+         application_form.family_name.present?
+      "#{application_form.given_names} #{application_form.family_name}"
```

Comment:
Minor thing: Does this need to be stripped of leading and trailing whitespace? It's possible one of the two names is not present.